web/concrete/css/build/core/app/panels/check-in.less

```diff
@@ -23,13 +23,81 @@ div#ccm-panel-check-in {
         border-bottom: 1px solid #464544;
         margin-bottom: 30px;
 
-        .fa-clock-o {
-            color: rgba(255,255,255,0.6);
-        }
         div.btn-group {
+            wdith: 100%;
```

Comment:
@a3020 @mlocati 

Hey guys,

Would it be ok to "lint" the css files, so mistakes like this one are detected on development?

We could use this: https://github.com/gruntjs/grunt-contrib-csslint